test/python/dialects/substrait/e2e_ibis.py

```diff
@@ -0,0 +1,36 @@
+# RUN: %PYTHON %s | FileCheck %s
+
+import ibis
+from ibis_substrait.compiler import core as ibis_ss
+
+from mlir_structured.dialects import substrait as ss
+from mlir_structured import ir
+
+
+def run(f):
+  print("\nTEST:", f.__name__)
+  with ir.Context(), ir.Location.unknown():
+    ss.register_dialect()
```

Comment:
We could probably move this to a test setup section, but also not terrible to do repeatedly here.